newIDE/electron-app/app/main.js

```diff
@@ -41,11 +41,13 @@ let mainWindow = null;
 const args = parseArgs(process.argv.slice(isDev ? 2 : 1), {
   // "Officially" supported arguments and their types:
   boolean: ['dev-tools', 'disable-update-check'],
-  string: '_', // Files are always strings
+  string: '_' // Files are always strings
 });
 
 // See registerGdideProtocol (used for HTML modules support)
-protocol.registerStandardSchemes(['gdide']);
+protocol.registerSchemesAsPrivileged([{ scheme: 'gdide' }]);
+
+app.allowRendererProcessReuse = true;
```

Comment:
Let's add a comment why we're doing this

Reply:
Oh yeah, forgot about that change :sweat_smile: . I'll add it to the PR description.

Reply:
I would also like a comment in the code so that it's obvious to the reader why there is this mysterious line here :)

Reply:
Ohh, I thought you meant the first line (`protocol.registerSchemesAsPrivileged`).

I'm not sure what comment I should add for the second line. Should I just say it's for avoiding the warning?

Reply:
Yes, let's go for something like "Set the option as its default value in future Electron versions."

Reply:
Okay, I added "Set to true, will be default value in future Electron versions".